Screen-Sharing-Video-Chat /js/app.js

```diff
@@ -0,0 +1,123 @@
+/* global OT API_KEY TOKEN SESSION_ID SAMPLE_SERVER_BASE_URL */
+
+let apiKey;
```

Comment:
We have to support IE, so (since we are not using babel or polyfills) we cannot use `let`. Please change all of these `let`s to `var`s.